configs/infer_config.yaml

```diff
@@ -17,12 +39,14 @@ label: 'inference_output'
 peaks: False
 tracks: False
 # Task is the task that model was trained for. If the task is regression, then model cannot
-# output denosied peaks. Similarly, if the task is classification, then model cannot 
+# output denoised peaks. Similarly, if the task is classification, then model cannot 
 # output denoised tracks. The above peaks, tracks options only are relevant if task is "both".
 task: "both"
 print_freq: 50
-bs: 512
+batch_size: 512
+# Number of workers to use for multiprocessing
 num_workers: 4
+# Path to model weights to be used for denoising
 weights_path: "None"
 # ID of the GPU to be used. Only relevant if running atacworks in single GPU mode.
 # If using multi-GPU mode i.e., if --distributed True, then this option is irrelevant. 
```

Comment:
Help with parameters should be all in one place so that users don't get confused about where to look for information. Currently, most of the help is in the help options of cmd_args.py so perhaps we should move this and other information there.

Reply:
Agree. removed from here. Added to the help